modules/http4s/build.sbt

```diff
@@ -0,0 +1,25 @@
+name := "pureconfig-http4s"
+
+libraryDependencies ++= Seq(
+  "org.http4s" %% "http4s-core" % "0.18.0",
+  Dependencies.scalaMacrosParadise,
+  Dependencies.scalaTest
+)
+
+crossScalaVersions := Seq("2.11.11", "2.12.4")
```

Comment:
Even though this works perfectly, it is not so convenient when we have to update Scala versions (with this we'd have to update in multiple places). Can you use something like we did with [the Akka module](https://github.com/pureconfig/pureconfig/blob/master/modules/akka/build.sbt#L4) instead?

Reply:
Ah, of course, that is better, will do.